yelp_bytes.py

```diff
@@ -59,3 +61,13 @@ def to_utf8(obj, errors='strict'):
 def from_utf8(obj, errors='strict'):
     """Decode utf8 bytes (str) to unicode text."""
     return from_bytes(obj, encoding='utf-8', errors=errors)
+
+
+def to_native(obj, errors='strict'):  # pragma: no cover
+    """ returns a native string regardless of py env """
+    if isinstance(obj, str):
+        return obj
+    elif PY2:
+        return obj.encode('UTF-8', errors)
+    else:  # PY3
+        return obj.decode('UTF-8', errors)
```

Comment:
Let's use from_bytes.

These simple changes makes this function edge-case free.